src/styles/custom.scss

```diff
@@ -21,3 +21,6 @@ body {
     margin-bottom: 50px;
 }
 
+.nav-link {
```

Comment:
We shouldn't add this to the custom css - add it to our GlobalStyles or locally to where `nav-link` is used in the header